File: mariomore.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <=0 || height > 8);
    
    for (int row = 1; row <= height; row++)
    {
        for (int space = height - row; space > 0; space--)
        {
            printf(" ");
        }
        for (int hash = 1; hash < row + 1; hash++)
        {
            printf("#");
        }
        
        printf("  ");
        for (int hash = 1; hash < row + 1; hash++)
        {
            printf("#");
        }
        printf("\n");   
    }
}

File: mariomore.py
# Felipe Rueda
# CS50 2019
# Pset 06

from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for row in range(height):
    print(" " * (height - (row + 1)), end="")
    print("#" * (row + 1), end="")
    print("  ", end="")
    print("#" * (row + 1), end="")
    print()
